### PR TITLE
Update and add missing beatmap statistic icons to info wedge

### LINKED ISSUE #2229: Add icons to beatmap info wedge
Web already has these icons. Spinner icon is not implemented yet tho.

### PR DESCRIPTION
**Pull request by peppy**
- [x] Depends on ppy/osu-resources#113
- [x] Depends on https://github.com/ppy/osu-resources/pull/114
- Closes https://github.com/ppy/osu/issues/2229

image

### CODE REVIEW

File: osu.Game/Screens/Select/BeatmapInfoWedge.cs [hunk 1]
```diff
@@ -418,10 +418,18 @@ public InfoLabel(BeatmapStatistic statistic)
                                     Anchor = Anchor.Centre,
                                     Origin = Anchor.Centre,
                                     RelativeSizeAxes = Axes.Both,
-                                    Scale = new Vector2(0.8f),
                                     Colour = Color4Extensions.FromHex(@"f7dd55"),
-                                    Icon = statistic.Icon,
+                                    Icon = FontAwesome.Regular.Circle,
```
Comment (smoogipoo) [inline osu.Game/Screens/Select/BeatmapInfoWedge.cs:422]: Isn't this only specifically for osu! rulesets? E.g. rush! now looks like:
image

File: osu.Game/Screens/Select/BeatmapInfoWedge.cs [hunk 2]
```diff
@@ -418,10 +418,18 @@ public InfoLabel(BeatmapStatistic statistic)
...
+                                    Scale = new Vector2(0.8f)
                                 },
+                                statistic.CreateIcon().With(i =>
+                                {
+                                    i.Anchor = Anchor.Centre;
+                                    i.Origin = Anchor.Centre;
+                                    i.RelativeSizeAxes = Axes.Both;
+                                    i.Size = new Vector2(1.2f);
```
[the remark this answers is not in the document]
Comment (smoogipoo) [reply osu.Game/Screens/Select/BeatmapInfoWedge.cs:430]: If we're going to go with this, at the very least this should be moved inside `BeatmapStatisticSprite` imo, and commented as to why it exists. I would expect this to be larger than the icon itself by reading the code.

[the remark this answers is not in the document]
Comment (peppy) [reply osu.Game/Screens/Select/BeatmapInfoWedge.cs:430]: There's no padding, no. It's to get the icons up to the correct size.

File: osu.Game/Screens/Select/BeatmapInfoWedge.cs (hunk 1 above)
Comment (peppy) [reply osu.Game/Screens/Select/BeatmapInfoWedge.cs:422]: Well that highly depends on expectation. Should the circle be always present or something optional? Probably need to figure that out and check with @arflyte.

File: osu.Game/Screens/Select/BeatmapInfoWedge.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (peppy) [reply osu.Game/Screens/Select/BeatmapInfoWedge.cs:430]: Turns out there is padding, as usual fucked assets provided to us. Will fix but requires a resources bump.

File: osu.Game/Screens/Select/BeatmapInfoWedge.cs (hunk 1 above)
Comment (peppy) [reply osu.Game/Screens/Select/BeatmapInfoWedge.cs:422]: I've adjusted the legacy icons to display inside the circle but also be sized correctly. Seems okay for now?

image

Comment (peppy) [reply osu.Game/Screens/Select/BeatmapInfoWedge.cs:422]: Some further tweaks to match flyte's originals better (except for circle thickness):

image

**Comment by smoogipoo** [comment]
lgtm :+1:

**Review by smoogipoo — DISMISSED** [review]

**Review by smoogipoo — DISMISSED** [review]

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**